docs/src/kubernetes/kind-values.yaml

```diff
@@ -0,0 +1,40 @@
+---
+zeebe-cluster: 
+  elasticsearch: 
+    imageTag: 6.8.3 
+# Permit co-located instances for solitary minikube virtual machines.
+    antiAffinity: "soft"
+
+# Shrink default JVM heap.
+    esJavaOpts: "-Xmx128m -Xms128m"
```

Comment:
These options are not recommended in a container I think, since `UseContainerSupport` is on by default in modern JVMs. Maybe something like
```
- name: JAVA_OPTS
  value: "-XX:MinRAMPercentage=60.0 -XX:MaxRAMPercentage=90.0"
...
resources: 
    limits:
        memory: 512Mi
    requests:
        memory: 256Mi
```
instead.

Reply:
That is Elastic Search recommendation, I will keep that for now

Reply:
they are in an old VM I think